metrics/ddl.go

```diff
@@ -34,14 +34,16 @@ var (
 			Buckets:   prometheus.ExponentialBuckets(0.01, 2, 20),
 		}, []string{LblType, LblResult})
 
-	BatchAddIdxHistogram = prometheus.NewHistogram(
+	BatchAddIndexSucc    = "batch_add_idx_succ"
```

Comment:
Can we just use `LblOK` and `LblError`?

Reply:
@coocood If we use `LblOK` and `LblError`, It causes redeclared.